test/acceptance/utils.spec.js

```diff
@@ -87,44 +87,19 @@ describe('Utils', function () {
         document.cookie = cname + '=' + cvalue + ';' + expires + ';path=/';
       };
 
-      var getCookie = function (cname) {
-        var name = cname + '=';
-        var ca = document.cookie.split(';');
-        for (var i = 0; i < ca.length; i++) {
-          var c = ca[i];
-          while (c.charAt(0) === ' ') {
-            c = c.substring(1);
-          }
-          if (c.indexOf(name) === 0) {
-            return c.substring(name.length, c.length);
-          }
-        }
-        return '';
-      };
-
-      setCookie('PathforaClosed_1', '1%257C1502732355490'); // double encoded
       setCookie('PathforaImpressions_2', '1|293847239874932871');
       sessionStorage.setItem('PathforaRecommend_2', '{"somejson": "here"}');
       setCookie('PathforaImpressions_3', '%badval%');
 
       pathfora.utils.updateLegacyCookies();
 
-      expect(getCookie('PathforaClosed_1')).toEqual('1%7C1502732355490');
```

Comment:
any reason, in particular, we removed the additional tests? overkill i assume?

Reply:
some of those assertions were concerned with double-encoding. At one point, there was a bug which was causing cookie values to be double-encoded, and then we had code for double-unencoding them to deal with cookies in the wild. Enough time has passed now (> the cookie expiration period) that we don't need to worry about cookies that were double-encoded, so we were able to remove the code and the tests. cc @sarahyablok 

More generally, keys and values stored in `localStorage` don't need to be URL encoded—localStorage manages to keep track of the content boundaries itself without cooperation from the API consumer. 😂

Reply:
:+1: